src/js/chat/handlers.js

```diff
@@ -523,6 +523,9 @@ var privmsg = exports.privmsg = function(channel, data) {
         from = data.bttvDisplayName;
     } else {
         from = helpers.lookupDisplayName(data.from);
+        if (bttv.settings.get('disableLocalizedNames') === true && from.toLowerCase() !== data.from) {
```

Comment:
I'd prefer you modify `lookupDisplayName` to disable these special display names for this case (need to allow nicknames but block twitch display names)